tests/unit/widget-filter-simple-condition.spec.ts

```diff
@@ -22,9 +23,17 @@ describe('Widget WidgetAggregation', () => {
   it('should have exactly 3 input components', () => {
     const wrapper = shallowMount(WidgetFilterSimpleCondition, { store: emptyStore, localVue });
     const autocompleteWrappers = wrapper.findAll('widgetautocomplete-stub');
-    expect(autocompleteWrappers.length).to.eql(2);
+    expect(autocompleteWrappers.length).to.equal(2);
     const inputtextWrappers = wrapper.findAll('widgetinputtext-stub');
-    expect(inputtextWrappers.length).to.eql(1);
+    expect(inputtextWrappers.length).to.equal(1);
+  });
+
+  it('should have exactly have a WidgetMultiInputText if operator is "in" or "nin"', async () => {
+    const wrapper = shallowMount(WidgetFilterSimpleCondition, { store: emptyStore, localVue });
+    wrapper.setData({ editedValue: { column: 'foo', value: [], operator: 'in' } });
+    await localVue.nextTick();
+    const autocompleteWrappers = wrapper.findAll('widgetmultiinputtext-stub');
```

Comment:
genuine question: is the `nextTick()` required here?

Reply:
Tested and it does not work...